exercises/practice/binary-search-tree/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+Insert and search for numbers in a binary tree.
+
+When we need to represent sorted data, an array does not make a good data structure.
+
+Say we have the array `[1, 3, 4, 5]`, and we add 2 to it so it becomes `[1, 3, 4, 5, 2]`.
+Now we must sort the entire array again!
+We can improve on this by realizing that we only need to make space for the new item `[1, nil, 3, 4, 5]`, and then adding the item in the space we added.
```

Comment:
I think this needs to be adapted to Elm to avoid the invalid code (list with multiple types), and the use of nil since a big part of Elm is the deliberate effort not to use it.

Reply:
This is a good point I think, and happens because this copy comes from the canonical source. I'm not even sure that line 9 adds much anyway, so maybe we could just remove it.

Reply:
We can't modify the instructions, they are standard for all tracks. At best, we can add an addendum, but I don't think it's worth it, many exercises are very vague, and in Exercism, tests are king.